docgen/pot/pot.go

```diff
@@ -0,0 +1,69 @@
+package pot
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+
+	"github.com/gunk/gunk/reflectutil"
+)
+
+const (
+	pot = `# SOME DESCRIPTIVE TITLE.
```

Comment:
I'd give this a better name - otherwise it's all very confusing, with a global `pot` in the file `pot.go` in the package `pot`